src/vmm/src/device_manager/mmio.rs

```diff
@@ -85,6 +87,7 @@ pub struct MMIODeviceManager {
     last_irq: u32,
     id_to_dev_info: HashMap<(DeviceType, String), MMIODeviceInfo>,
     raw_io_handlers: HashMap<(DeviceType, String), Arc<Mutex<dyn RawIOHandler>>>,
+    block_devices: HashMap<String, Arc<Mutex<Block>>>,
```

Comment:
In the end, are these all going to be a single:
`devices: HashMap<String, Arc<Mutex<dyn VirtioDevice>>>` ?

Reply:
We are going to use Alex's commit right [here](https://github.com/alexandruag/firecracker/commit/d00c572ba4404be866c7b2394c75d9fc6dd22054)  to get rid of this temporary hashtable.